R/ggplotly.R

```diff
@@ -836,7 +836,7 @@ gg2list <- function(p, width = NULL, height = NULL,
           }
           # facets have multiple axis objects, but only one title for the plot,
           # so we empty the titles and try to draw the title as an annotation
-          if (nchar(axisTitleText) > 0) {
+          if (!is.null(axisTitleText) && nchar(axisTitleText) > 0) {
```

Comment:
```suggestion
          if (nzchar(axisTitleText %||% "")) {
```

Reply:
Interestingly, this variant fails the test, even though it definitely returns `FALSE` when passed a `NULL` or `logical(0)` explicitly. Not sure what's different about how `axisTitleText` is being received. For now I'll get it passing the test.

Reply:
Totally forgot about `rlang::is_empty` for dealing with this stuff. That seems to do the trick.

Reply:
plotly currently doesn't have an rlang dependency, so I'd rather use `length(x) == 0` than `rlang::is_empty()`

Reply:
`rlang` was already in the imports for `eval_tidy`: https://github.com/ropensci/plotly/blob/master/DESCRIPTION#L43